.eslintrc

```diff
@@ -11,7 +11,8 @@
   ],
   "env": {
     "browser": true,
-    "node": true
+    "node": true,
+	"mocha": true
```

Comment:
Wrong indentation level

Reply:
Looks like this was due to tabs vs. spaces as well